Resources/Prototypes/Reagents/narcotics.yml

```
@@ -223,6 +223,32 @@
   plantMetabolism:
   - !type:PlantAdjustHealth
     amount: -5
+  metabolisms: # SS220 Cigarettes rework
```

Аналогично с комментарием выше
```suggestion
  metabolisms: # SS220 Cigarettes rework
```